runtime/runtime.c

```diff
@@ -253,9 +253,35 @@ void* allocate_n_bytes_ptr(u32 n) {
 WEAK void populate_globals() {}
 
 // Code that actually runs the wasm code
-IMPORT i32 wasmf_main(i32 a, i32 b);
+IMPORT void wasmf__start(void);
+
+int runtime_argc = 0;
+
+char *runtime_argv_buffer = NULL;
+int runtime_argv_buffer_len = 0;
+
+void runtime_cleanup() {
+    // Cancel any pending timeout
```

Comment:
Remove the commented out code?

Reply:
This is previously commented out code that Gregor wrote. I just shifted the cleanup part here to make sure that it runs via atexit. I figure we should defer figuring out the utility of this code until later, so keep it commented out for now.

Reply:
Do we want timeout support in aWsm?

Reply:
I can't imagine we would in runtime.c. Maintaining time/using setjmp etc... are all platform specific. Thoughts?

Reply:
I defer to Gabe on this one, it was just an experiment, and we can always revive it later

Reply:
Yeah, I can't think of a specific target use case. Since we'll have it in the commit history, I'll remove the commented out code.